src/main/java/tictactoe/PlayerTypes.java

```diff
@@ -2,7 +2,8 @@
 
 public enum PlayerTypes {
```

Comment:
If the Enum is: PlayerType, then the type signature will read better wherever it's used.

Reply:
Thanks, I'll change this